examples/server/server.cpp

```diff
@@ -2132,6 +2134,15 @@ static void server_params_parse(int argc, char **argv, server_params &sparams,
             }
             sparams.hostname = argv[i];
         }
+        else if (arg == "--http-cors-origin")
+        {
+            if (++i >= argc)
+            {
+                invalid_param = true;
+                break;
+            }
+            sparams.http_cors_origin = argv[i];
```

Comment:
Is it possible here to input a list of domain names, for example `example.com,mywebsite.com,example.net` ?

Reply:
Done. It also made for a pretty convenient way to include both "localhost" and "127.0.0.1" by default.